src/reducers/reducer.js

```diff
@@ -8,7 +8,9 @@ const initialState = {
   allUsers: [],
   isLoggedIn: true,
   user: {},
-  resultsSwitch: 'hangouts'
+  resultsSwitch: 'hangouts',
+  showProfile: false,
+  currentProfile: {}
```

Comment:
currentProfile holds the object of the selected user from the "Hangouts" component